bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs

```diff
@@ -40,21 +62,21 @@ public SystemUserClient(HttpClient httpClient)
         return null;
     }
 
-    public async Task<SystemUserReal?> PostNewSystemUserReal(SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default)
+    public async Task<SystemUserReal?> PostNewSystemUserReal(
```

Comment:
Waht does Real mean here

Reply:
In the BFF we have descriptions of new SystemUsers coming in from the Frontend with the SystemUserDescriptor Model. Returning is the SystemUserReal from the Authentication component. So Real is used as a suffix to mean that it exists on the db in the Authentication component.